gradle.properties

```diff
@@ -1,8 +1,8 @@
 # The version of qdrant to use to download protos
-qdrantProtosVersion=v1.8.0
+qdrantProtosVersion=dev
 
 # The version of qdrant docker image to run integration tests against
-qdrantVersion=v1.8.0
+qdrantVersion=dev
```

Comment:
TODO: Switch to `v1.9.0` when out.

Reply:
Done in 970739388fec958bb7e16fc7e19d2d1688cd1d49